### PR TITLE
Add FormCreator documentation

### PR DESCRIPTION
**Pull request by trasher**
Backport from GH wiki: https://github.com/pluginsGLPI/formcreator/wiki

### CODE REVIEW

**Comment by trasher** [comment]
Visible at: http://glpi-plugins.readthedocs.io/en/feature-formcreator/formcreator/

File: source/formcreator/installation.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+.. warning::
+
+   The plugin's directory must have the same name as the plugin:
+
+     * **Good**: `glpi/plugins/formcreator`
+     * **Bad**: `glpi/plugins/formcreator-master`
+     * **Bad**: `glpi/plugins/formcreator-0.90-1.3.2`
+
+   Only one directory must contains the plugin's files of a single plugin in the GLPI plusins directory. **Don't rename the plugin's directory for backup, move it!**
```
Comment (orthagh) [inline source/formcreator/installation.rst]: plusins

File: source/formcreator/configuration.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+Based on question type options will be based on type chosen.
+
+If validation is desired, it can be implemented following `PHP Regular Expressions <http://php.net/manual/reference.pcre.pattern.syntax.php>`_.
+
+Helpdesk
+^^^^^^^^
+
+The plugin can provide its own design for helpdesk.
+
+To enable it, edit an entity, open tne **Forms** tab and set the field **Use Form creator's helpdesk** to **Yes**. This setting handled inheritance from parent entity to children entities.
```
Comment (orthagh) [inline source/formcreator/configuration.rst]: tne -> the

File: source/formcreator/configuration.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+Users using the simplified interface will benefit a new interface allowing them to
```
Comment (orthagh) [inline source/formcreator/configuration.rst]: missing :

**Pull request merged**